docs/sources/writing-guide/front-matter/index.md

```diff
@@ -44,14 +44,11 @@ The following headings describe what each element does and provides guidelines f
 
 ### aliases
 
-Provides an HTML redirect from the pages in the list to the current page.
-For more information, refer to [Hugo aliases](#hugo-aliases).
+Used to create redirects from the previous URL to the new URL when a page changes or moves. As a best practice, when you rename or move files, you should create an alias with the previous file path and name to create an HTML redirect from the old URL to the new URL. For more information, refer to [Hugo aliases](#hugo-aliases).
```

Comment:
Aliases use URL paths, not file paths.

I also chose to remove the specific mention of HTML redirects because we should revise that information. The actual production website does not use HTML redirects but HTTP redirects. 
However, the local webserver still uses HTML redirects.
In practice, they result in the same behavior.
 
```suggestion
Used to create redirects from the previous URL to the new URL when a page changes or moves. 
As a best practice, when you rename or move files, you should create an alias with a reference to the previous URL path create a redirect from the old URL to the new URL. For more information, refer to [Hugo aliases](#hugo-aliases).
```

Reply:
Is this a best practice or a requirement? What's the scenario where you've renamed or moved a page and you don't want an alias?

Reply:
I guess I just want stronger language. Maybe talk both scenarios. If it's possible for there to be a working redirect (you move an entire file or rename it) set up a redirect. If you've deleted content and there's no corresponding content or it's moved to within another page then don't. I just don't want people left with the idea that redirects are a nice-to-have. They're a requirement unless you can't set one up feasibly or you're gonna create bad 404s.

Reply:
Ah, gotcha.  Let me think about rewording.  We definitely need **_something_**, because currently we don't say anything about why you would need an alias, and we definitely want to encourage people to create them.